app/react/Viewer/components/Viewer.js

```diff
@@ -28,11 +28,12 @@ import {
 import { openPanel } from '../actions/uiActions';
 import { selectDoc } from '../selectors';
 import ConfirmCloseForm from './ConfirmCloseForm';
-import SourceDocument from './SourceDocument';
-import TargetDocument from './TargetDocument';
+
 import ViewMetadataPanel from './ViewMetadataPanel';
 import ViewerDefaultMenu from './ViewerDefaultMenu';
 import ViewerTextSelectedMenu from './ViewerTextSelectedMenu';
+import SourceDocument from './SourceDocument';
```

Comment:
why is this import order change needed ? does this has something to do with circular dependencies ?

Reply:
no, just that at first I did those loadables, and then I changed it back, and ended in a different order